ovos_persona/__init__.py

```
@@ -324,7 +330,67 @@ def match_high(self, utterances: List[str], lang: Optional[str] = None,
+                elif self.voc_match(query, "summon", lang=closest_lang):
+                    for name in self.personas:
+                        if name.lower() in query:
+                            query = self.remove_voc(query, "summon", lang=closest_lang)
+                            query = self.remove_voc(query, "persona", lang=closest_lang)
+                            match = {"name": "summon.intent",
+                                     "conf": 0.85,
+                                     "entities": {"persona": name, "query": query}}
+                            break
+
+            name =  match.get("name")
+            conf =  match.get("conf", 0)
```

Ruff (0.8.2)

Local variable `conf` is assigned to but never used

Remove assignment to unused variable `conf`

(F841)